### PR TITLE
open-cluster-management/backlog#2667 netw 20

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @TheRealHaoLiu . We started this and got some content out for 1.x. I would like to also push it to 2.0, if it is still accurate. If not, can we work on updating it for 2.0.x and 2.1? Thanks!

File: install/requirements.adoc [hunk 1]
```diff
@@ -37,3 +37,23 @@ See the following versions that are tested and supported:
...
 See the https://access.redhat.com/articles/5248271[{product-title} {product-version} Support matrix] for additional information.
+
+[#Network configuration]
+= Network configuration
+
+Configure your network settings to allow the following connections:
+
+Hub cluster:
+* Outbound connectivity to cloud provider's API
+* Outbound connectivity to the Kubernetes API server of theprovisioned `ManagedCluster` on port 6443
```
Comment (TheRealHaoLiu) [inline install/requirements.adoc]: theprovisioned

Comment (cadawson) [reply install/requirements.adoc]: Updated. Thanks.

**Comment by cadawson** [comment]
Updated @TheRealHaoLiu . Can we roll this out? Thanks.

**Review by TheRealHaoLiu — APPROVED** [review]

**Pull request merged**